### PR TITLE
Replaced default db with Postgres

### PR DESCRIPTION
**Pull request by yaourtv**
To install Postgres on your local machine use
**Ubuntu [Debian/Mint/ElementaryOS/any -buntu]:**
`sudo apt install postgresql postgresql-contrib libpq-dev`
**Arch [Manjaro/Anarchy/Archlabs]:**
`sudo pacman -S postgresql postgresql-libs`

Creating user is pretty similar on every distro, so just check links below and use preferred one:
[Digital Ocean: Ubuntu 14.04](https://www.digitalocean.com/community/tutorials/how-to-use-postgresql-with-your-ruby-on-rails-application-on-ubuntu-14-04)
[GitHub: ArchLinux](https://github.com/malnvenshorn/OctoPrint-FilamentManager/wiki/Setup-PostgreSQL-on-Arch-Linux)

About user in database.yml:
I'm not sure that fields like _username_ and _password_ should be revealed since it's security-sensitive information (literally database superuser login info) so I'm gonna ask what to do with them on Telegram chat soon:tm:

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -1,25 +1,25 @@
-# SQLite version 3.x
-#   gem install sqlite3
+# PostgreSQL. Versions 8.2 and up are supported.
 #
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem 'sqlite3'
+# Configure Using Gemfile
+# gem 'pg'
 #
-default: &default
```
Comment (lnvch) [inline config/database.yml]: I'd suggest to leave default section, just replace adapter and other fields here
don't see production section, you won't be able to deploy to heroku without production section
you can see example here:
https://devcenter.heroku.com/articles/getting-started-with-rails5#add-the-pg-gem

Comment (yaourtv) [reply config/database.yml]: @lnvch fixed. I saw info that heroku uses its own production db so it's not necessary to have this section in database.yml
However, added production section too.

File: config/database.yml [hunk 2]
```diff
@@ -1,25 +1,32 @@
...
 
 production:
   <<: *default
-  database: db/production.sqlite3
+  adapter: postgresql
+  encoding: unicode
+  database: notes_project_test
+  pool: 5
+  username: dbadmin
+  password: DBAdmin
```
Comment (lnvch) [inline config/database.yml]: It's bad idea to store password from your db in github, would be a security issue
You could use approach from heroku guides:
https://devcenter.heroku.com/articles/getting-started-with-rails5#add-the-pg-gem

Comment (lnvch) [inline config/database.yml]: It's bad idea to store password from your db in github, would be a security issue
You could use approach from heroku guides:
https://devcenter.heroku.com/articles/getting-started-with-rails5#add-the-pg-gem

File: config/database.yml [hunk 3]
```diff
@@ -1,25 +1,32 @@
...
 default: &default
-  adapter: sqlite3
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
 
 development:
-  <<: *default
-  database: db/development.sqlite3
-
+  adapter: postgresql
```
Comment (lnvch) [inline config/database.yml]: the same stuff from all the envs, could be added to default section, no need in it otherwise

**Comment by lnvch** [comment]
@yaourtv two more comments here

**Comment by yaourtv** [comment]
@lnvch done. Hope that no more changes are required.

**Pull request merged**